hobbify_stats/model.py

```diff
@@ -0,0 +1,38 @@
+#!/usr/bin/env python
+"""This script contains the necessary method to consult the score of a date and update it"""
+import querys
+
+def relative_freq(username, date):
+    """ This function returns a number between 0 and 1 that is named
+        relative frequency. After,  realize a conversion to numbers between 0 and 4
+        for use in the  heatmap development in ReactJS
+    """
+
+    total = querys.count_intances_pending(username, date)[0]
+    done = querys.count_intances_done(username, date)[0]
+    value = 0
+    if total != 0:
+        score = done / total
+        if score == 0:
+            value = 0
+        elif  0 < score <= 0.25:
+            value = 1
+        elif  0.25 < score <= 0.5:
+            value = 2
+        elif  0.5 < score <= 0.75:
+            value = 3
+        else:
+            value = 4
```

Comment:
Lower bounds doesn't have to be checked. Just check for upper bounds like:

```python
if score == 0:
elif score <= 0.25:
elif score <= 0.5:
elif score <= 0.75:
else:
```

Actually, you just need to get the integer part (or the ceiling) of `score * 4` since `score` is within the range `[0, 1]`